core/src/main/java/org/pentaho/di/core/database/BaseDatabaseMeta.java

```diff
@@ -518,6 +519,23 @@ public void setServername( String servername ) {
     this.servername = servername;
   }
 
+  /**
+   * @return Returns the warehouse.
+   */
+  @Override
```

Comment:
Is there a way to avoid new get/setWarehouse methods?  It's not applicable to any other db that I'm aware of, so it stinks to have to add it to the base.